### DISCUSSION TITLE
Open in new tab shortcut

### DISCUSSION THREAD

**Discussion by Muizzyranking** [zen-browser/desktop#2940] [Q&A]
When I hit ctrl + click on a link to open a new tab, it opens a pop up and then I would have to click the maximise button to maximise.

there are some times I would want to open multiple links and I usually find myself using right click and choosing the open in new tab option. Is there a new how can i change this behaviour because I use vimium and the open in new tab works the same way as ctrl + click

**Comment by Rasalas** [comment]
imho a feature like that should not overwrite well established shortcuts as Ctrl+Click.  
I think Arc does this for `target="_blank"` links and that usually feels pretty natural.

You can click links with MOUSE 3 (usually the scrolling wheel button) and they'll open in a new tab.  
Shift+MOUSE 3 to jump to it immediately.

You can also disable that feature or change the Trigger method in Settings -> Look and Feel -> Glance
